phidgets_analog_outputs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>phidgets_analog_outputs</name>
+  <version>2.1.2</version>
+  <description>Driver for the Phidgets Analog Output devices</description>
+
+  <maintainer email="martin.guenther@dfki.de">Martin Günther</maintainer>
+  <maintainer email="clalancette@openrobotics.org">Chris Lalancette</maintainer>
+  <maintainer email="robotique@capacites.fr">Alexis Fetet</maintainer>
```

Comment:
If you're adding yourself as a maintainer for the `phidgets_analog_output` package, please watch this repository so that you can actually answer questions by other users, test pull requests, triage bugs etc. about the `phidgets_analog_output` package.